aws/README.md

```diff
@@ -1,9 +1,22 @@
 # AWS
 
-As of this writing, we support two types of AWS deployments with Cluster API:
-- A "standard" cluster with HA control plane and a machine deployment for workers
-- A cluster that uses autoscaling groups for the worker set.
-Autoscaling groups are called "MachinePools" in Cluster API lingo.
+As of this writing, we support two types of AWS deployments with Cluster API (CAPI):
+
+- A "standard" cluster with HA control plane and a machine deployment for workers.
+- A cluster that uses autoscaling groups for the worker set. Autoscaling groups are called "MachinePools" in Cluster API lingo.
+
+We also provide different variants for Talos 0.10 and up that added better support for
+[external cloud providers](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/),
+including [cloud-provider-aws](https://github.com/kubernetes/cloud-provider-aws).
+
+| Directory                                                       | Talos | cluster-api-provider-aws (CAPA) |
+| --------------------------------------------------------------- | ----- | ------------------------------- |
+| [standard](./standard/)                                         | 0.9+  | 0.5.4                           |
+| [standard-external](./standard-external/)                       | 0.9+  | 0.5.4                           |
+| [autoscaling-workers](./autoscaling-workers/)                   | 0.10+ | 0.6.1+                          |
+| [autoscaling-workers-external](./autoscaling-workers-external/) | 0.10+ | 0.6.1+                          |
+
+- cloud-provider-aws: v1.20.0-alpha.0
```

Comment:
Should we mention it and k8s versions?

Should we test Talos with different versions on CI?